src/fmu/dataio/readers.py

```diff
@@ -0,0 +1,112 @@
+"""Module for reading data not supported by other tools"""
+
+from __future__ import annotations
+
+import hashlib
+import json
+from dataclasses import dataclass, field
+from pathlib import Path
+
+
+def read_faultroom_file(filename: str | Path) -> FaultRoomSurface:
+    """Faultroom surface data are (geo)JSON file or dicts; needs separate handling.
+
+    Faultroom data are quite propriatary data and is not supported by e.g. xtgeo
+    currently. Hence it is read locally. As input, both a dict and a file with
+    extension .json og .geojson can be applied
+    """
+    filename = Path(filename)
+    if "json" in filename.suffix.lower():
+        # try read the (geo)json file
+        with open(filename, "r", encoding="utf-8") as stream:
+            dict_obj = json.load(stream)
```

Comment:
Potential talking point: the builtin json parser is a bit "slow" compared to parsers like https://docs.rs/jiter/latest/jiter/ . Given the size of these files, it might be worth a bit of looking into (at a later stage)

Reply:
We can take it later